lib/content-services/src/lib/document-list/components/filter-header/filter-header.component.spec.ts

```diff
@@ -109,7 +109,37 @@ describe('FilterHeaderComponent', () => {
         expect(setCurrentRootFolderIdSpy).toHaveBeenCalled();
     });
 
-    it('should set active filters when an initial value is set', async () => {
+    it('should set filters if initial value is provided', async () => {
+        spyOn(queryBuilder, 'setCurrentRootFolderId');
+        spyOn(queryBuilder, 'isCustomSourceNode').and.returnValue(false);
+        spyOn(queryBuilder, 'setActiveFilter');
+
+        const initialFilterValue = { name: 'pinocchio' };
```

Comment:
no need for that const because you use it only in next line so you can assign it directly to value